src/ArmTemplates/Common/API/Clients/Abstractions/ApiClientBase.cs

```diff
@@ -0,0 +1,52 @@
+﻿// --------------------------------------------------------------------------
+//  <copyright file="ApiClientBase.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation. All rights reserved.
+//  </copyright>
+// --------------------------------------------------------------------------
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Extensions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Utilities;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.API.Clients.Abstractions
+{
+    public abstract class ApiClientBase
+    {
+        public static string BaseUrl = "https://management.azure.com";
+
+        protected Authentication Auth = new Authentication();
```

Comment:
Just like elsewhere. Can these be properties instead?

Reply:
changed it